optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -71,160 +69,27 @@ image::Chapter-Quick_start/cloudOptimizationValueProposition.png[align="center"]
 [[cloudBalancingDomainModelDesign]]
 === Domain Model Design
 
-Beginning with the domain model:
 
+Using a <<domainModelingGuide,domain model>> helps determine which classes are planning entities and which of their properties are planning variables. Using a domain model helps to simplify contraints, improve performance, and increase flexibility for future needs. 
+
+//procedure from chapter 17 - Design Patterns
```

Comment:
Part of the modularization strategy is to have everything the user needs to know to complete a task in one place. If we send them off to Chapter 17 to learn all about Design Patterns, we might not get them back. It's only a short part of the procedure that I've borrowed from there, so I'm hoping we can leave it in.